_includes/working-with-tables.md

```diff
@@ -0,0 +1,208 @@
+1. Delete the value in the top row of the _Position_ column. Fill down the rest of the _Position_ column so that all cells are empty.
+1. Click **Fill Boxes by Column**.
+    * The _Position_ column values will change. Note the new pattern of the positions.
+
+<a name="check-qcs"/>
+
+### Check QCs (Libraries only)
+The _Check QCs_ feature can be used to automatically update the _QC Passed?_ column values based on the values in the _Size (bp)_, _Volume_, or _Conc._ columns.
+
+1. Return to your _Create Libraries_ bulk table (or [click here](https://miso.gsi.oicr.on.ca/miso/library/bulk/receive?quantity=5&projectId=&sampleClassId=15) if you have navigated away from the page).
+1. Ensure all _QC Passed?_ cells are set to **Unknown**.
+1. In the top row, enter the following values:
+    * _Size (bp)_: **300**
+    * _Volume_: **10**
+    * _Conc._: **2**
+1. In the second row, enter the following values:
+    * _Size (bp)_: **301**
+    * _Volume_: **11**
+    * _Conc._: **3**
+1. Use the [incrementing fill](#fill-down-incrementing) to fill in the _Size (bp)_, _Volume_, and _Conc._ columns.
+    * Note that the _Vol. Units_ column does not contain numbers and so cannot be incrementally filled down.
+1. Click **Check QCs**.
+1. In the _Size_ section, select **>** from the dropdown menu and enter **303** into the input field.
+1. Click **Check**.
+    * Note that the _QC Passed?_ values have changed.
+1. Click **Check QCs**.
+1. In the _Volume_ section, select **=** from the dropdown menu and enter **12** into the input field.
+1. Click **Check**.
+    * Note that the _QC Passed?_ values have changed.
+1. Click **Check QCs**.
+1. In the _Concentration_ section, select **<=** from the dropdown menu and enter **3** into the input field.
+1. Click **Check**.
+    * Note that the _QC Passed?_ values have changed.
+
+<a name="export"/>
+
+### Export
+The current state of the bulk table data can be exported at any time. Exports can be saved as Microsoft Excel, Open Document Format, or Comma-delimited Data (CSV) files.
+
+1. Return to your _Create Libraries_ bulk table (or [click here](https://miso.gsi.oicr.on.ca/miso/library/bulk/receive?quantity=5&projectId=&sampleClassId=15) if you have navigated away from the page).
+1. Click **Export Data** at the top of the table.
+1. Select **Microsoft Excel** format.
+1. Click **Export**.
+    * The spreadsheet will begin downloading.
+
+<a name="import"/>
+
+### Import
+A spreadsheet can be uploaded to a bulk table page, and any matching data from the spreadsheet will be entered in the table.
+    * In order to match, the spreadsheet column header must exactly match the bulk table column header.
+    * Spreadsheet upload will fail if:
+        * the spreadsheet contains columns whose headers don't match any headers in the bulk table
+        * the spreadsheet contains more rows than the bulk table does
+    * If the spreadsheet cells contain invalid data, the data will still be entered in the table.
```

Comment:
These bullet points aren't working for some reason. I think the first level shouldn't be indented